Index: src/main/java/com/example/mahjong/LoginController.java
===================================================================
diff --git a/src/main/java/com/example/mahjong/LoginController.java b/src/main/java/com/mahjong/client/LoginController.java
rename from src/main/java/com/example/mahjong/LoginController.java
rename to src/main/java/com/mahjong/client/LoginController.java
--- a/src/main/java/com/example/mahjong/LoginController.java	
+++ b/src/main/java/com/mahjong/client/LoginController.java	
@@ -1,4 +1,4 @@
-package com.example.mahjong;
+package com.mahjong.client;
 
 import javafx.fxml.FXML;
 import javafx.fxml.FXMLLoader;
Index: src/main/java/com/example/mahjong/Client.java
===================================================================
diff --git a/src/main/java/com/example/mahjong/Client.java b/src/main/java/com/mahjong/client/Client.java
rename from src/main/java/com/example/mahjong/Client.java
rename to src/main/java/com/mahjong/client/Client.java
--- a/src/main/java/com/example/mahjong/Client.java	
+++ b/src/main/java/com/mahjong/client/Client.java	
@@ -1,22 +1,30 @@
-package com.example.mahjong;
+package com.mahjong.client;
 
 import javafx.application.Application;
+
 import javafx.fxml.FXMLLoader;
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 
 import java.io.IOException;
+import java.net.Socket;
 
 public class Client extends Application {
+    //当玩家点击加入游戏与服务端建立通道 ok
+    //两个线程：一个接受服务端的消息，更新画布 ok
+    //另一个向服务端发消息，玩家操作消息 ok
+    //玩家点击相应操作时，就向服务端发消息
 
     @Override
     public void start(Stage stage) throws IOException {
-        FXMLLoader fxmlLoader = new FXMLLoader(Client.class.getResource("login.fxml"));
-        Scene scene = new Scene(fxmlLoader.load(), 600, 400);
+        FXMLLoader fxmlLoader = new FXMLLoader(Client.class.getResource("menu.fxml"));
+        Scene scene = new Scene(fxmlLoader.load(), 1000, 800);
         stage.setTitle("Mahjong");
         stage.setScene(scene);
         stage.show();
+
     }
+
 
     public static void main(String[] args) {
         launch();
Index: src/main/resources/com/example/mahjong/login.fxml
===================================================================
diff --git a/src/main/resources/com/example/mahjong/login.fxml b/src/main/resources/com/mahjong/client/login.fxml
rename from src/main/resources/com/example/mahjong/login.fxml
rename to src/main/resources/com/mahjong/client/login.fxml
--- a/src/main/resources/com/example/mahjong/login.fxml	
+++ b/src/main/resources/com/mahjong/client/login.fxml	
@@ -10,7 +10,7 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.scene.text.Text?>
 
-<VBox alignment="CENTER" spacing="10" fx:controller="com.example.mahjong.LoginController" xmlns:fx="http://javafx.com/fxml">
+<VBox alignment="CENTER" spacing="10" fx:controller="com.mahjong.client.LoginController" xmlns:fx="http://javafx.com/fxml">
   <Text text="游戏登录" />
   <GridPane hgap="10" vgap="10" alignment="CENTER">
     <Label text="用户名:" GridPane.columnIndex="0" GridPane.rowIndex="0" />
Index: src/main/resources/com/example/mahjong/menu.fxml
===================================================================
diff --git a/src/main/resources/com/example/mahjong/menu.fxml b/src/main/resources/com/mahjong/client/menu.fxml
rename from src/main/resources/com/example/mahjong/menu.fxml
rename to src/main/resources/com/mahjong/client/menu.fxml
--- a/src/main/resources/com/example/mahjong/menu.fxml	
+++ b/src/main/resources/com/mahjong/client/menu.fxml	
@@ -3,10 +3,9 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.scene.text.Text?>
 
-<VBox alignment="CENTER" fx:controller="com.example.mahjong.MenuController" spacing="20" xmlns:fx="http://javafx.com/fxml">
+<VBox alignment="CENTER" fx:controller="com.mahjong.client.MenuController" spacing="20" xmlns:fx="http://javafx.com/fxml">
     <Text text="麻将游戏" />
-    <Button text="创建房间" fx:id="createRoomButton" onAction="#handleCreateRoom" />
-    <Button text="加入房间" fx:id="joinRoomButton" onAction="#handleJoinRoom" />
+    <Button text="开始游戏" fx:id="gameStartButton" onAction="#handleGameStart" />
     <Button text="游戏规则" fx:id="gameRulesButton" onAction="#handleGameRules" />
     <Button text="退出游戏" fx:id="exitGameButton" onAction="#handleExitGame" />
 </VBox>
